training/views.py

```diff
@@ -24,58 +25,80 @@
 
 from .models import Training
 
+logger = logging.getLogger(__name__)
+
 
 def register(request):
-    if request.method != "POST":
-        # if a GET (or any other method) we'll create a blank form
+    host = request.META.get("HTTP_HOST", "localhost")
```

Comment:
previously this was [designed to exit early](https://medium.com/swlh/return-early-pattern-3d18a41bba8), to decrease necessary indentation / makes it easier to figure out where the else branch starts.

I'm not sure this feels more readable? Do you have an argument for doing it this way?

Reply:
I don't have a strong opinion really, but return once is the [Django way of doing it](https://docs.djangoproject.com/en/2.1/topics/forms/#the-view). I think it's easier to follow the request/response logic with fewer returns, and it's DRYer when the return statement is identical.